particle-builtins/src/builtins.rs

```diff
@@ -756,12 +757,12 @@ where
         let dependencies = Args::next("dependencies", &mut args)?;
         let blueprint_request = AddBlueprint { name, dependencies };
 
-        let blueprint_request = serde_json::to_value(blueprint_request).map_err(|err| {
+        let blueprint_request = blueprint_request.to_string().map_err(|err| {
             JError::new(format!(
-                "Error serializing blueprint_request to JSON: {err}"
+                "Error serializing blueprint_request to IPLD: {err}"
```

Comment:
I know it's an old code, but let's use the chance to actualise it a little bit

```suggestion
                "Error serializing blueprint to IPLD: {err}"
```